distributed/deploy/tests/test_spec_cluster.py

```diff
@@ -197,25 +195,20 @@ async def test_unexpected_closed_worker(cleanup):
             assert len(cluster.workers) == 2
 
 
-@pytest.mark.flaky(reruns=10, reruns_delay=5)
-@pytest.mark.slow
-@pytest.mark.asyncio
-async def test_restart(cleanup):
-    # Regression test for https://github.com/dask/distributed/issues/3062
+@gen_test(timeout=60)
```

Comment:
slow marker should probably still be here, shouldn't it? increased timeout sounds like slow

Reply:
increased timeout is just for exceptional cases on Windows, where the CI frequently finds itself gasping for air. On a typical laptop, this test completes in 1.5s.